lib/launcher.js

```diff
@@ -173,76 +134,54 @@ var Launcher = function (server, emitter, injector) {
     'config.processKillTimeout'
   ]
 
-  this.kill = function (id, callback) {
-    var browser = getBrowserById(id)
-    callback = callback || function () {}
+  this.kill = (id, callback = () => {}) => {
+    const browser = getBrowserById(id)
 
-    if (!browser) {
-      process.nextTick(callback)
-      return false
+    if (browser) {
+      browser.forceKill().then(callback)
+      return true
     }
+    process.nextTick(callback)
+    return false
 
-    browser.forceKill().then(callback)
-    return true
   }
 
-  this.restart = function (id) {
-    var browser = getBrowserById(id)
-
-    if (!browser) {
-      return false
+  this.restart = (id) => {
+    const browser = getBrowserById(id)
+    if (browser) {
+      browser.restart()
+      return true
     }
-
-    browser.restart()
-    return true
+    return false
   }
 
-  this.killAll = function (callback) {
+  this.killAll = (callback = () => {}) => {
     log.debug('Disconnecting all browsers')
 
-    var remaining = 0
-    var finish = function () {
-      remaining--
-      if (!remaining && callback) {
-        callback()
-      }
-    }
-
-    if (!self._browsers.length) {
+    if (!this._browsers.length) {
       return process.nextTick(callback)
     }
 
-    self._browsers.forEach(function (browser) {
-      remaining++
-      browser.forceKill().then(finish)
-    })
+    Promise.all(
+      this._browsers
+        .map((browser) => browser.forceKill())
+    ).then(callback)
   }
 
-  this.areAllCaptured = function () {
-    return !self._browsers.some(function (browser) {
-      return !browser.isCaptured()
-    })
+  this.areAllCaptured = () => {
+    return this._browsers.every((browser) => browser.isCaptured())
```

Comment:
Can we use an expression return:
`() => this._browsers.every((browser) => browser.isCaptured())`
?

Reply:
Good point, fixed!